linode/provider.go

```diff
@@ -57,6 +57,7 @@ func Provider() terraform.ResourceProvider {
 			"linode_sshkey":                 dataSourceLinodeSSHKey(),
 			"linode_user":                   dataSourceLinodeUser(),
 			"linode_volume":                 dataSourceLinodeVolume(),
+			"linode_lke":                    dataSourceLinodeLKE(),
```

Comment:
This looks great so far! Just one thought at first glance: LKE includes clusters, kubeconfigs, API server endpoints, and node pools. It might make more sense to call this resource `linode_lke_cluster`, the same way `linode_nodebalancer_config` is used for NodeBalancers.

Reply:
Thanks! It's a good point, I was debating myself the naming, I'll make that change and update the PR.